pkg/service/grpc_service.go

```diff
@@ -26,9 +28,12 @@ type GRPCService struct {
 	Logger *log.Entry
 }
 
+// Serve allows for the use of GRPC only without HTTPP, where as HTTP service enables both
+// GRPC and http
 func (s *GRPCService) Serve(ctx context.Context, eval eval.IEvaluator) error {
 	s.Eval = eval
 
+	// TODO: Needs TLS implementation
```

Comment:
Can you create an issue for this TODO?

Reply:
Done https://github.com/open-feature/flagd/issues/103